services/madoc-ts/src/frontend/admin/pages/content/manifests/edit-manifest-metadata.tsx

```diff
@@ -43,6 +43,21 @@ export const EditManifestMetadata: UniversalComponent<EditManifestMetadataType>
 
       await api.updateManifestMetadata(Number(params.id), diff);
 
+      const manifest = await api.getManifestById(Number(params.id));
+
+      const contexts = await api.getManifestProjects(Number(params.id));
+      console.log(contexts);
+      const resource = {
+        id: params.id,
+        thumbnail: manifest.manifest.thumbnail || '',
+        // how do I map here to the right shape for the resource
+        resource: { ...manifest.manifest, type: 'Manifest', id: params.id },
+        contexts: { contexts },
+      };
+
+      // // Reingest the search index
+      await api.searchReIngest(resource);
+
```

Comment:
This will be called from the frontend - we could do the same step in the `update-metadata` on the backend - which would catch cases when the metadata it updated from the API